repos/spack_repo/builtin/packages/py_tokenizers/package.py

```diff
@@ -24,7 +25,8 @@ class PyTokenizers(PythonPackage):
 
     # TODO: This package currently requires internet access to install.
```

Comment:
FYI. The archive from `PyPi.org` indicates the minimum python for the new version is `python@3.9:`.